packages/deprecation-crawler/src/lib/commands/default.command.ts

```diff
@@ -8,11 +8,11 @@ import { commitChanges } from '../tasks/commit-changes';
 import { getVersion, run } from '../utils';
 import { CrawledRelease } from '../models';
 import { YargsCommandObject } from '../cli/model';
-import { DEFAULT_COMMAND_NAME } from '../cli';
 import { crawl } from '../processors/crawl';
 
 export const defaultCommand: YargsCommandObject = {
-  command: DEFAULT_COMMAND_NAME,
+  // * means the default command
```

Comment:
Where is the default defined now?

Reply:
`*` is the default command. https://github.com/yargs/yargs/blob/master/docs/advanced.md#default-commands

When no command is passed, or when the passed command doesn't exist it will use this command.
We don't have to do anything for this to work, this is all built into yargs.